### ISSUE TITLE
New feature request

### ISSUE THREAD

**Issue by reijerteunis** [spencerccf/app_settings#145]
Hi,

Thanks for this amazing plugin. I have a question, can you also add functionality to open the health settings.

Thanks.

Ruud

**Comment by plinkos** [comment]
I believe you will need to add health kit permissions and such to your app and then make native request to prompt user for access/permissions and then they should appear in the app settings screen from then on.

**Comment by reijerteunis** [comment]
Yes but once you already used the health kit permission you cannot do it again. Therefor I wanted to use your module to redirect the user directly to the native setting.
